src/App.tsx

```diff
@@ -1,24 +1,23 @@
-import { useState } from 'react'
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { AuthContextProvider } from "./contexts/AuthContext";
+// import "./index.css";
+import SignUpPage from "./Components/SignUpPage";
+import SignInPage from "./Components/SignInPage";
+import "./index.css";
 
 function App() {
-  const [count, setCount] = useState(0)
-
   return (
-    <>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
+    <BrowserRouter>
+      <main>
+        <AuthContextProvider>
+          <Routes>
+            <Route path="/signUp" element={<SignUpPage />} />
+            <Route path="/signIn" element={<SignInPage />} />
```

Comment:
there was not requirement for sign in page or route this should be removed

Reply:
removed sign in page